source/docs/contracts_deployment.md

```diff
@@ -64,6 +64,30 @@ module.exports = {
 }
 </code></pre>
 
+
+### Node accounts + user accounts
+
+If you still wish to use your Ethereum node' account(s) along with your custom accounts, just use the following:
+
+<pre><code class="javascript">// config/contracts.js
+module.exports = {
+  "testnet": {
+      "deployment": {
+        "accounts": [
+          <mark class="highlight-inline">{
+            "nodeAccounts": true
+          }</mark>,
+          {
+            // One of you own account
+          }
+        ]
+      }
+  }
+}
+</code></pre>
+
+The order in the `accounts` array is important. So putting `nodeAccounts` as first, as above, will make the Node's account as the `defaultAccount` for deployment.
```

Comment:
I'd rewrite this to:

The order in the `accounts` array is important. This means that using `nodeAccounts` first, as above, will set the node's account as the `defaultAccount` for deployment.